lib/storage/index.js

```diff
@@ -85,10 +85,15 @@ module.exports = function(env) {
       pre.notNull(encryption);
       pre.notNull(privatePair);
 
-      var decipher = crypto.createDecipher(encryption, this.calculateEncryptionKey(privatePair));
-      decipher.write(value);
-      decipher.end();
-      return decipher.read().toString();
+      try {
+        var decipher = crypto.createDecipher(encryption, this.calculateEncryptionKey(privatePair));
+        decipher.write(value);
+        decipher.end();
+        return decipher.read().toString();
+      } catch (err) {
+        log.error('Failed to decrypt value using specified encryption and private pair', err);
+        return null;
```

Comment:
The calling code (also in this PR), checks to see if the return value exists.  If it does not exist, then it fails gracefully (as opposed to dying horribly, which is want it does before this PR).